packages/babel-parser/src/plugins/flow.js

```diff
@@ -759,6 +765,7 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       allowExact: boolean,
       allowSpread: boolean,
       allowProto: boolean,
+      allowInexact: boolean,
```

Comment:
Aside: we should use an object instead of all these boolean params

Reply:
(y) can do!